app/domain/authentication/authn_jwt/authentication_parameters.rb

```diff
@@ -0,0 +1,22 @@
+module Authentication
+  module AuthnJwt
+    # Data class to store data regarding jwt token that is needed during the jwt authentication process
+    class AuthenticationParameters
+      attr_accessor :decoded_token, :jwt_token
+      attr_reader :authenticator_name, :service_id, :account, :username, :client_ip, :request
+
+      def initialize(authentication_input:, jwt_token:)
+        @authenticator_name = authentication_input.authenticator_name
+        @service_id = authentication_input.service_id
+        @account = authentication_input.account
+        @username = authentication_input.username
+        @client_ip = authentication_input.client_ip
```

Comment:
please consider using `def_delegators` and `Forwardable` instead of setting all these explicitly

Reply:
```
extend(Forwardable)
def_delegators(:@authenticator_input, :account, :username, :client_ip, :authenticator_name, :service_id)

def initialize(authentication_input:, jwt_token:)        
  @authenticator_input = authentication_input
  @jwt_token = jwt_token
end
```

Reply:
Didnt work. Lets review this tommorow out of scope of this PR